recipe/meta.yaml

```diff
@@ -9,21 +9,18 @@ package:
 source:
   fn: {{ filename }}
   url: https://github.com/{{ name }}/{{ name }}/archive/{{ filename }}
-  sha256: bb43216b075b93b767e6cbaecd5e95d087da887bcdc51afd36d940d8bd9f9819
+  sha256: bdbf9a2e6988526bfeb8c56ce3cdfe2998d60ac289078e2215374288185e8c8d
 
 build:
   number: 0
   # We skip s390x because it's not supported.
-  skip: True # [s390x]
-  # py-boost hasn't been built for Python 3.11 yet.
-  skip: True # [py<37 or py>310]
+  # py-boost 1.73 isn't available for Python 3.11 on Windows.
```

Comment:
can we try with latest Visual Studio ( is it 2019, I suppose ) ?

Reply:
It's a good suggestion. I'll try

Reply:
@M-Waszkiewicz-Anaconda A new error with vs2019:
```
    conda_build.exceptions.OverLinkingError: overlinking check failed

    ['  ERROR (rdkit,Lib/site-packages/rdkit/Chem/Scaffolds/rdScaffoldNetwork.pyd): $RPATH/boost_serialization.dll not found in packages, sysroot(s) nor the missing_dso_whitelist.\n.. is this binary repackaging?', '  ERROR (rdkit,Library/lib/RDKitFileParsers.dll): $RPATH/boost_iostreams.dll not found in packages, sysroot(s) nor the missing_dso_whitelist.\n.. is this binary repackaging?', '  ERROR (rdkit,Library/lib/RDKitFilterCatalog.dll): $RPATH/boost_serialization.dll not found in packages, sysroot(s) nor the missing_dso_whitelist.\n.. is this binary repackaging?', '  ERROR (rdkit,Library/lib/RDKitmaeparser.dll): $RPATH/boost_iostreams.dll not found in packages, sysroot(s) nor the missing_dso_whitelist.\n.. is this binary repackaging?', '  ERROR (rdkit,Library/lib/RDKitSubstructLibrary.dll): $RPATH/boost_serialization.dll not found in packages, sysroot(s) nor the missing_dso_whitelist.\n.. is this binary repackaging?', '  ERROR (rdkit,Library/bin/RDKitScaffoldNetwork.dll): $RPATH/boost_serialization.dll not found in packages, sysroot(s) nor the missing_dso_whitelist.\n.. is this binary repackaging?', '  ERROR (rdkit,Lib/site-packages/rdkit/Chem/rdSubstructLibrary.pyd): $RPATH/boost_serialization.dll not found in packages, sysroot(s) nor the missing_dso_whitelist.\n.. is this binary repackaging?', '  ERROR (rdkit,Library/bin/RDKitGraphMol.dll): $RPATH/boost_serialization.dll not found in packages, sysroot(s) nor the missing_dso_whitelist.\n.. is this binary repackaging?', '  ERROR (rdkit,Library/lib/RDKitScaffoldNetwork.dll): $RPATH/boost_serialization.dll not found in packages, sysroot(s) nor the missing_dso_whitelist.\n.. is this binary repackaging?', '  ERROR (rdkit,Library/bin/RDKitmaeparser.dll): $RPATH/boost_iostreams.dll not found in packages, sysroot(s) nor the missing_dso_whitelist.\n.. is this binary repackaging?', '  ERROR (rdkit,Library/lib/RDKitRDStreams.dll): $RPATH/boost_iostreams.dll not found in packages, sysroot(s) nor the missing_dso_whitelist.\n.. is this binary repackaging?', '  ERROR (rdkit,Library/bin/RDKitRDStreams.dll): $RPATH/boost_iostreams.dll not found in packages, sysroot(s) nor the missing_dso_whitelist.\n.. is this binary repackaging?', '  ERROR (rdkit,Lib/site-packages/rdkit/Chem/rdChemReactions.pyd): $RPATH/boost_serialization.dll not found in packages, sysroot(s) nor the missing_dso_whitelist.\n.. is this binary repackaging?', '  ERROR (rdkit,Library/bin/RDKitSubstructLibrary.dll): $RPATH/boost_serialization.dll not found in packages, sysroot(s) nor the missing_dso_whitelist.\n.. is this binary repackaging?', '  ERROR (rdkit,Lib/site-packages/rdkit/Chem/rdfiltercatalog.pyd): $RPATH/boost_serialization.dll not found in packages, sysroot(s) nor the missing_dso_whitelist.\n.. is this binary repackaging?', '  ERROR (rdkit,Library/lib/RDKitChemReactions.dll): $RPATH/boost_serialization.dll not found in packages, sysroot(s) nor the missing_dso_whitelist.\n.. is this binary repackaging?', '  ERROR (rdkit,Library/bin/RDKitTautomerQuery.dll): $RPATH/boost_serialization.dll not found in packages, sysroot(s) nor the missing_dso_whitelist.\n.. is this binary repackaging?', '  ERROR (rdkit,Library/bin/RDKitFilterCatalog.dll): $RPATH/boost_serialization.dll not found in packages, sysroot(s) nor the missing_dso_whitelist.\n.. is this binary repackaging?', '  ERROR (rdkit,Library/bin/RDKitChemReactions.dll): $RPATH/boost_serialization.dll not found in packages, sysroot(s) nor the missing_dso_whitelist.\n.. is this binary repackaging?', '  ERROR (rdkit,Library/lib/RDKitGraphMol.dll): $RPATH/boost_serialization.dll not found in packages, sysroot(s) nor the missing_dso_whitelist.\n.. is this binary repackaging?', '  ERROR (rdkit,Library/lib/RDKitTautomerQuery.dll): $RPATH/boost_serialization.dll not found in packages, sysroot(s) nor the missing_dso_whitelist.\n.. is this binary repackaging?', '  ERROR (rdkit,Library/bin/RDKitFileParsers.dll): $RPATH/boost_iostreams.dll not found in packages, sysroot(s) nor the missing_dso_whitelist.\n.. is this binary repackaging?']
```

I'll update `missing_dso_whitelist` as:
```
  missing_dso_whitelist:
    - '*/RDKit*dll'                     # [win]
    - '$RPATH/boost_serialization.dll'  # [win]
    - '$RPATH/boost_iostreams.dll'      # [win]
```

Reply:
Hmm, it fails to import python modules:
```
Traceback (most recent call last):

  File "C:\b\abs_1fca69bz5t\croot\rdkit_1696328200118\test_tmp\run_test.py", line 8, in <module>

    import rdkit.Chem

  File "C:\b\abs_1fca69bz5t\croot\rdkit_1696328200118\_test_env\lib\site-packages\rdkit\Chem\__init__.py", line 16, in <module>

    from rdkit.Chem import rdchem

ImportError: DLL load failed while importing rdchem: The specified module could not be found.
```

Building the package on `win64` against `libboost/py-boost 1.73` is more reliable but without **py311** support. I would revert my latest changes and I'll build with libboost 1.73